test.sh

```diff
@@ -1,43 +1,24 @@
 #!/bin/bash -ex
 
-test_image=api-test
-docker build -t $test_image .
-
 function finish {
-	docker rm -f $pg_cid
-	docker rm -f $server_cid
+  docker-compose down
```

Comment:
Can you add `-v` to this, so it removes any created volumes as well? Better safe than sorry, eats up CI build space.